src/k8s_sandbox/_sandbox_environment.py

```diff
@@ -178,6 +184,24 @@ async def read_file(self, file: str, text: bool = True) -> str | bytes:
                     temp_file.read() if not text else temp_file.read().decode("utf-8")
                 )
 
+    async def connection(self) -> SandboxConnection:
+        return SandboxConnection(type="k8s", command="")
+
+    async def run_dropbear(self) -> None:
+        with open(
+            Path(__file__).parent / "resources" / "dropbear", "rb"
+        ) as dropbear_script:
+            await self.write_file("/usr/bin/dropbear", dropbear_script.read())
```

Comment:
We spoke offline re being able to pre-install openssh server, but presuming this approach sticks, a hardlink would be marginally faster + less space

Reply:
(I don't think a hardlink would work. The dropbear binary is on one k8s pod, and this command is copying it to another pod (the sandbox environment).)